verify-node/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+extern crate wasm_bindgen;
+
+use ed25519_dalek::Signature;
+use holo_config_core::AdminPublicKey;
+use std::str::FromStr;
+use wasm_bindgen::prelude::*;
+
+// Use `wee_alloc` as the global allocator.
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```

Comment:
Only meaningful for browser. Even for browser, this only saves a web kilobytes, i.e. < 0.5% size, while making allocation performance significantly worse.

Reply:
Agreed. Once again, taken from previous code example.  If it makes a small difference, I think we should get rid of them all, to decrease the complexity and reduce the risk of our code.